config/locales/en.yml

```diff
@@ -1420,7 +1420,7 @@ en:
       case/overturned_ico/foi:
         sub_heading: Create an ICO Overturned (FOI) case
       offender_sar:
-        rejected: Create Rejected Offender SAR case
+        rejected: Create rejected offender SAR case
```

Comment:
I think Offender should definitely be capitalised

Reply:
Shall I confirm with Alan as he suggested offender?